src/plugins/flare/flareplugin.cpp

```diff
@@ -290,6 +294,7 @@ bool FlarePlugin::write(const Tiled::Map *map, const QString &fileName)
     out << "tilewidth=" << map->tileWidth() << "\n";
     out << "tileheight=" << map->tileHeight() << "\n";
     out << "orientation=" << orientationToString(map->orientation()) << "\n";
+    out << "backgroundColorARGB=" << backgroundColor.name(QColor::HexArgb) << "\n";
```

Comment:
I'd prefer if you just named this `backgroundColor`. The format `#AARRGGBB` is so common that I don't see a good reason to include that in the name.

Apart from that, note that Tiled can also open these files, but I see no code related to reading back the background color property.